src/collection/collectionview.cpp

```diff
@@ -561,6 +565,97 @@ void CollectionView::OpenInNewPlaylist() {
 
 }
 
+void CollectionView::SearchForThis() {
+
+  QModelIndex current = currentIndex();
+  QVariant type = model()->data(current, CollectionModel::Role_Type);
+  if (!type.isValid() || (type.toInt() != CollectionItem::Type_Song && type.toInt() != CollectionItem::Type_Container && type.toInt() != CollectionItem::Type_Divider)) {
+    return;
+  }
+  QString search;
+  QModelIndex index = qobject_cast<QSortFilterProxyModel*>(model())->mapToSource(current);
+
+  switch (type.toInt()) {
+    case CollectionItem::Type_Song:{
+      SongList songs = app_->collection_model()->GetChildSongs(index);
+      if (!songs.isEmpty()) {
+        last_selected_song_ = songs.last();
+      }
+      search = QString("title:%1").arg(last_selected_song_.title());
+      break;
+    }
+
+    case CollectionItem::Type_Container:
+    case CollectionItem::Type_Divider:{
```

Comment:
I think we should ignore `CollectionItem::Type_Divider`, divider is the letter above the first group by level, testing this here now, it only prefixes `albumartist:` with no text since it's trying to get the metadata for ie.: artist or albumartist for the divider too. I think we should just ignore the divider, so move `CollectionItem::Type_Divider` to it's own case with a break.